.cirrus.yml

```diff
@@ -1002,3 +1163,30 @@ release_test_task:
     clone_script: *get_gosrc
     setup_script: *setup
     main_script: *main
+
+
+# When preparing to release a new version, this task may be manually
+# activated at the PR stage to verify the build is proper for a potential
+# podman release.
+#
+# Note: This cannot use a YAML alias on 'release_task' as of this
+# comment, it is incompatible with 'trigger_type: manual'
+release_test_aarch64_task:
+    name: "Optional Release Test {aarch64}"
+    alias: release_test_aarch64
+    # Release-PRs always include "release" or "Bump" in the title
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: $CIRRUS_CHANGE_TITLE =~ '.*((release)|(bump)).*'
+    # Allow running manually only as part of release-related builds
+    # see RELEASE_PROCESS.md
+    trigger_type: manual
+    depends_on:
+        - build_aarch64
+        - success
+    ec2_instance: *standard_build_ec2_aarch64
+    env:
+        <<: *stdenvars_aarch64
+        TEST_FLAVOR: release
+    clone_script: *get_gosrc_aarch64
+    setup_script: *setup
+    main_script: *main
```

Comment:
This task isn't strictly needed.  Really it's just checking `version.go` details, which is not architecture or platform dependent.  I'd recommend just leaving this whole section out.

Reply:
removed.